### ISSUE TITLE
'height' not provided in processing NGSIM dataset

### ISSUE THREAD

**Issue by zbzhu99** [huawei-noah/SMARTS#1145]
## **BUG REPORT**
**High Level Description**
When creating SMARTS with NGSIM scenarios, it will raise a `sqlite3.OperationalError` with `no such column: height` when reading traffic history data.

**SMARTS version**
current develop branch

**Details**
I suspect the problem is related to changing the SQL command here:

https://github.com/huawei-noah/SMARTS/blob/e3681e7a04b7f4cc448525900fe65b5f76415bcb/smarts/core/traffic_history.py#L123

to

```python
query = "SELECT length, width, height, type FROM Vehicle WHERE id = ?"
```

in the develop branch. However, NGSIM traffic history generator does not save `height` to the database file (*.shf):

https://github.com/huawei-noah/SMARTS/blob/3d69ba1f5b9995ee8b12638676d92e5f4750f3a5/smarts/sstudio/genhistories.py#L353-L362

**Error logs and screenshots**
```
  File "/NAS2020/Workspaces/DRLGroup/zbzhu/MADPO/SMARTS_Imitation/bug_reproduce_zbzhu.py", line 117, in __init__
    self._next_scenario()
  File "/NAS2020/Workspaces/DRLGroup/zbzhu/MADPO/SMARTS_Imitation/bug_reproduce_zbzhu.py", line 196, in _next_scenario
    self.vehicle_missions = self.scenario.discover_missions_of_traffic_histories()
  File "/NAS2020/Workspaces/DRLGroup/zbzhu/MADPO/SMARTS_Imitation/SMARTS/smarts/core/scenario.py", line 434, in discover_missions_of_traffic_histories
    veh_length, veh_width, veh_height = self._traffic_history.vehicle_size(v_id)
  File "/NAS2020/Workspaces/DRLGroup/zbzhu/MADPO/SMARTS_Imitation/SMARTS/smarts/core/traffic_history.py", line 138, in vehicle_size
    tuple, query, params=(vehicle_id,)
  File "/NAS2020/Workspaces/DRLGroup/zbzhu/MADPO/SMARTS_Imitation/SMARTS/smarts/core/traffic_history.py", line 65, in _query_val
    cur.execute(query, params)
sqlite3.OperationalError: no such column: height
```

**Comment by sah-huawei** [comment]
@RutvikGupta Could you please take a look at this?  (I think this is due to some changes that were made for the Waymo dataset back in August.)

**Comment by sah-huawei** [comment]
@zbzhu99 The `.shf` format was updated in August to add a `height` column to the database file, even for NGSIM.  This requires regenerating the `.shf` file.  You can do so with:
`scl scenario build --clean scenarios/ngsim`

If you do this, you will probably also want the new map (attached to Issue #969) and `.yaml` file to go with it.  I will attach the `.yaml` file here next.

**Comment by RutvikGupta** [comment]
https://github.com/huawei-noah/SMARTS/blob/3d69ba1f5b9995ee8b12638676d92e5f4750f3a5/smarts/sstudio/genhistories.py#L146

`.shf` files were updated to add height as default if they don't exist.
![image](https://user-images.githubusercontent.com/47334253/144357311-f9ce6df9-0432-417b-b3a0-0cce3548cbe1.png)
@zbzhu99 you may want to rebuild the scenario as mentioned by @sah-huawei  above

**Comment by sah-huawei** [comment]
I'm attaching some example `.yaml` files that I used with the new map (attached to Issue #969) to generate the new `.shf` files.

If you put these into your ngsim scenario folder, change the `input_path` line in each to point to the appropriate trajectory file in your filesystem, and then do `scl scenario build --clean scenarios/ngsim`, it should generate the new `.shf` files to go with the new map.
[i80_yaml.zip](https://github.com/huawei-noah/SMARTS/files/7639107/i80_yaml.zip)

Alternatively, I can generate them here for you and send them to you again like we did last time if you prefer.  (They are big files.)
